packages/expo-local-authentication/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ _This version does not introduce any user-facing changes._
 
 - Avoid LAContext#biometryType bug on iOS 11.0.0. ([#12413](https://github.com/expo/expo/pull/12413) by [@mickamy](https://github.com/mickamy/))
 - Do not require activity existence on getKeyguardManager. ([#12400](https://github.com/expo/expo/pull/12400) by [@mickamy](https://github.com/mickamy/))
+- Flip flag on cancel authentication. ([#12728](https://github.com/expo/expo/pull/12728) by [@mickamy](https://github.com/mickamy/))
```

Comment:
```suggestion
- Flip `isAuthenticating ` when the user dismisses the authentication dialog on Android. ([#12728](https://github.com/expo/expo/pull/12728) by [@mickamy](https://github.com/mickamy/))
```

Or something similar to this 😉

Reply:
@lukmccall Thanks, I followed your suggestion!